polls/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
 
 # Create your models here.
+class Question(models.Model):
+    # adding a string method
+    def __str__(self):
+        return self.question_text
+    question_text = models.CharField(max_length=200)
+    pub_date = models.DateTimeField('date published')
+
+class Choice(models.Model):
+    # adding 
+        question = models.ForeignKey(Question, on_delete=models.CASCADE)
+        choice_text = models.CharField(max_length=200)
+        votes = models.IntegerField(default=0)
```

Comment:
```suggestion
    votes = models.IntegerField(default=0)
```

Let's outdent these three lines a little bit. Example above.